### DISCUSSION TITLE
difference between github wiki and the documention

### DISCUSSION THREAD

**Discussion by matan-h** [helix-editor/helix#2067] [Q&A] [answered]
There are some things (like "How to install the default language servers") that are only in [GitHub wiki](https://github.com/helix-editor/helix/wiki).
And some things (like "Adding Textobject Queries" that are only in [Documention](https://docs.helix-editor.com/)) .
And they also have similar pages ("keymap").
What the difference between the two?

**Comment by the-mikedavis** [comment, marked as answer]
GitHub Wikis can be edited by anyone which make them appropriate for fast-changing documents like the language-server installation wiki ([here](https://github.com/helix-editor/helix/wiki/How-to-install-the-default-language-servers)). By default, documentation tends to go into the documentation website for new features and configuration.

The keymaps document on the documentation website serves as a reference of the current set of keybinds. The [keymap brainstorm](https://github.com/helix-editor/helix/wiki/Keymap-brainstorm) wiki is some collaborative editing by users and should not be used as a reference. It is a wiki for people ideating on the current and future states of the keymap. There's also a keymap discussion [issue](https://github.com/helix-editor/helix/issues/165).

**Reply by matan-h** [reply]
can we link "How to install the default language servers" in the wiki from documentation, and so on?

**Reply by the-mikedavis** [reply]
Yep! Actually there's an open PR for it: https://github.com/helix-editor/helix/pull/2065

**Reply by sudormrfbin** [reply]
It was added in https://github.com/helix-editor/helix/pull/2037, but it seems like it only changed the readme and a section for new language authors. An addition to the installation section of the docs would indeed be welcome then.

Edit: Whoops seems like I replied too soon :)
